modules/telco-core-gitops-operator-and-ztp-plugins.adoc

```diff
@@ -43,15 +44,16 @@ Since {product-title} 4.15, the `spec.extraManifestPath` field is deprecated.
 ====
 
 Engineering considerations::
-* Set the `MachineConfigPool` (`mcp`) CR `paused` field to true during a cluster upgrade maintenance window and set the `maxUnavailable` field to the maximum tolerable value.
+* Set the `MachineConfigPool` (`MCP`) CR `paused` field to true during a cluster upgrade maintenance window and set the `maxUnavailable` field to the maximum tolerable value.
 This prevents multiple cluster node reboots during upgrade, which results in a shorter overall upgrade.
 When you unpause the `mcp` CR, all the configuration changes are applied with a single reboot.
 +
 [NOTE]
 ====
-During installation, custom `mcp` CRs can be paused along with setting `maxUnavailable` to 100% to improve installation times.
+During installation, custom `MCP` CRs can be paused along with setting `maxUnavailable` to 100% to improve installation times.
```

Comment:
i don't see the note about having no CNFs running in this case, per https://gitlab.cee.redhat.com/reference-configurations/reference-design-specifications/-/merge_requests/69 - was that intended to be internal only?  cc @imiller0